source/features/copy-file.tsx

```diff
@@ -42,7 +42,7 @@ function removeButton(): void {
 function init(): void {
 	delegate('.rgh-copy-file', 'click', handleClick);
 
-	if (select.exists('.blob.instapaper_body')) {
+	if (select.exists('.blob.js-code-block-container')) {
```

Comment:
I'd prefer something more self-explanatory
```suggestion
	if (select.exists('.blob > .markdown-body')) {
```